src/services/codefixes/importFixes.ts

```diff
@@ -720,7 +720,9 @@ namespace ts.codefix {
 
             // check exports with the same name
             const exportSymbolWithIdenticalName = checker.tryGetMemberInModuleExportsAndProperties(symbolName, moduleSymbol);
-            if (exportSymbolWithIdenticalName && checkSymbolHasMeaning(exportSymbolWithIdenticalName, currentTokenMeaning)) {
+            if (exportSymbolWithIdenticalName && checkSymbolHasMeaning(exportSymbolWithIdenticalName, currentTokenMeaning)
+                // Don't auto-import a method of an `export =` module (See GH#20191)
+                && !(exportSymbolWithIdenticalName.flags & SymbolFlags.Method)) {

```

Comment:
this is going to break many of the dt definitions.. for instance, JQuery: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/jquery/index.d.ts#L28

Reply:
Instead of just relying on the meaning, I think what's more correct is to check is if the symbol's declarations is part of the global `Object`, `String`, `Number` , or `Symbol` declaration, right? I think you could probably check using our current internal checker APIs by doing something like
```ts
exportSymbolWithIdenticalName.valueDeclaration && checker.getPropertyOfType(checker.getStringType(), exportSymbolWithIdenticalName.name).valueDeclaration === exportSymbolWithIdenticalName.valueDeclaration // repeat for number, symbol, object
```